client/signup/steps/domains/index.jsx

```diff
@@ -476,7 +476,7 @@ class DomainsStep extends React.Component {
 				isEligibleVariantForDomainTest={ this.isEligibleVariantForDomainTest() }
 				suggestion={ initialQuery }
 				designType={ this.getDesignType() }
-				vendor={ getSuggestionsVendor( true ) }
+				vendor={ getSuggestionsVendor( ! this.props.isDomainOnly ) }
```

Comment:
This might be overkill, but one thing that came to mind was that it could be helpful to change the declaration of the `getSuggestionsVendor` function from taking a boolean (which results in cryptic code like `getSuggestionsVendor( true )`) to taking an object with a clearly named property. Like this:
```
function getSuggestionsVendor( { isSignup: false } ) {}
```

Then the invocations would be easier to understand: `getSuggestionsVendor( { isSignup: ! this.props.isDomainOnly } )`. It should be a small change since there's basically two places where we actually explicitly pass anything to this function, but I totally get if you don't want to do it as part of this PR.